src/components/profile/DataClassification.vue

```diff
@@ -69,12 +91,26 @@ export default {
           return this.fluent('classification_info');
       }
     },
+    isVolunteerWithSlackAccess() {
+      return (
+        !this.staffInformation.staff.value &&
+        'slack-access' in (this.accessInformation.mozilliansorg.values || {})
+      );
+    },
+    slackLabel() {
+      return 'Slack';
```

Comment:
I seemed urge to add a function for that because the `pre-commit` hook `husky` was denying a hardcoded span label at all cost... And I didn't think this is something which can be localized.

Reply:
I see. In that case it should live in there as data (move to it the `data()` method / create one here).